controllers/storageclient_controller.go

```diff
@@ -433,14 +433,24 @@ func (s *StorageClientReconciler) reconcileClientStatusReporterJob(instance *v1a
 	cronJob.Namespace = s.OperatorNamespace
 	addLabel(cronJob, storageClientNameLabel, instance.Name)
 	addLabel(cronJob, storageClientNamespaceLabel, instance.Namespace)
+	var jobDeadLineSeconds int64 = 155
+	var podDeadLineSeconds int64 = 120
+	var keepJobResourceSeconds int32 = 600
+	var reducedKeptSuccecsful int32 = 1 
```

Comment:
Had to create passable variables to the pointer variables required in the new fields. looking into other options as well.